konf/site.yaml

```diff
@@ -204,6 +204,9 @@ production:
     if ($host = 'cloud.ubuntu.com' ) {
       rewrite ^ https://ubuntu.com/public-cloud$request_uri? permanent;
     }
+    if ($host = 'certification.ubuntu.com' ) {
+      rewrite ^ https://ubuntu.com/certification$request_uri? permanent;
```

Comment:
We'll also need to redirect all the old URL structures to their new pages.

Reply:
The queries have been done in the view and the URL paths have been changed in the `redirects.yaml`.